packages/vscode/src/index.ts

```diff
@@ -33,7 +33,7 @@ export async function activate(ext: ExtensionContext) {
 
   const ctx = (Array.isArray(root) && root.length)
     ? await rootRegisterManual(ext, root, projectPath, status)
-    : await rootRegisterAuto(ext, typeof root === 'string' ? root : projectPath, config, status)
+    : await rootRegisterAuto(ext, typeof root === 'string' ? root : projectPath, config, status, projectPath)
```

Comment:
Should we resolve root in the second argument directly?

Reply:
sure